### PR TITLE
Lidor/feat/preference bar

### PR DESCRIPTION
**Pull request by lidor12456**

### CODE REVIEW

File: src/components/Nav.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import { PrimaryNav, MenuLink, Menu } from "./NavElements";
+import { constants } from "../routes/constants";
+import chatIcon from "../assets/imgs/chat-icon.png";
+function Nav() {
+  const [userPicture, setUserPicture] = useState(
+    "https://www.w3schools.com/howto/img_avatar.png"
```
Comment (AmeerLalaDeveloper) [inline src/components/Nav.js:7]: u can download this image and save it in assets file

File: src/pages/volunteer page/components/CheckboxesTree/CheckboxesTree.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <>
+      <CheckboxTree
+        nodes={regionsData}
+        expanded={expanded}
+        onExpand={(expanded) => setExpanded(expanded)}
+        checked={checked}
+        onCheck={(checked) => setChecked(checked)}
+        showNodeIcon={false}
+      />
+      {console.log(`check: ${checked}`)}
```
Comment (AmeerLalaDeveloper) [inline src/pages/volunteer page/components/CheckboxesTree/CheckboxesTree.js]: remove this console

File: src/pages/volunteer page/components/CheckboxesTree/styles.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+.list-container {
+  /* width: 30vw;
+  height: 20vh; */
+}
+
+ul {
```
Comment (AmeerLalaDeveloper) [inline src/pages/volunteer page/components/CheckboxesTree/styles.css]: please check with meir if the default tags style in same file (styles.css) i think he did it in base.css or something

File: src/pages/volunteer page/components/PersonalData/PersonalData.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import * as S from "./PersonalData.style";
+
+const DataContainer = styled.div`
```
Comment (AmeerLalaDeveloper) [inline src/pages/volunteer page/components/PersonalData/PersonalData.js:4]: u can delete this u arleady used it in styled components

File: src/components/Nav.js (hunk 1 above)
Comment (lidor12456) [reply src/components/Nav.js:7]: This is not the real pic. the pic will come from the backend so I prefer to save it on state right now

File: src/pages/volunteer page/components/PlaceCard/PlaceCard.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import styled from "styled-components";
+import chatIcon from "../../../../assets/imgs/chat-icon.png";
+import vIcon from "../../../../assets/imgs/Vicon.png";
+import xIcon from "../../../../assets/imgs/Xicon.png";
+import * as S from "./PlaceCard.style";
+
+function PlaceCard(props) {
```
Comment (AmeerLalaDeveloper) [inline src/pages/volunteer page/components/PlaceCard/PlaceCard.js:9]: Destruct  properties directly ,({ placeId,placePic .....})

File: src/pages/volunteer page/components/PlaceCard/PlaceCard.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        </S.ImageContainer>
+      </Link>
+      <S.BtnsContainer>
+        <S.Btns>
+          <S.ButtonsImg src={xIcon} alt="X icon" />
+        </S.Btns>
+        <S.Btns
+          onClick={() => {
+            if (isClicked) {
+              navigate("/chat");
```
Comment (AmeerLalaDeveloper) [inline src/pages/volunteer page/components/PlaceCard/PlaceCard.js:27]: use the constant variable u have

File: public/index.html [hunk 7]
```diff
@@ -3,6 +3,7 @@
   <head>
```
Comment (AmeerLalaDeveloper) [inline public/index.html:3]: files structure :
1- in the Prefences bar  folder.. 
u dont have component called Preferences bar so u dont need this folder ,extract the components inside and delete folder 

2- Days and Regions  folders should be named as component name

**Pull request closed without merging**